dali/base/dadfs.cpp

```diff
@@ -138,19 +138,22 @@ static StringBuffer &normalizeFormat(StringBuffer &in)
     return in;
 }
 
-static StringBuffer &getAttrQueryStr(StringBuffer &str,const char *sub,const char *key,const char *name)
+static StringBuffer &getAttrQueryStr(StringBuffer &str,const char *sub,const char *key,const char *name, bool nameCaseInsensitive)
 {
     assertex(key[0]=='@');
-    str.appendf("%s[%s=\"%s\"]",sub,key,name);
+    if (nameCaseInsensitive)
+        str.appendf("%s[%s=?\"%s\"]",sub,key,name);
```

Comment:
as long as these are not top-level properties, and the only one nameCaseInsnsitive=true is used for is for Protect/@name, then the loss of efficiency this implies is okay I think.